src/tests/unit/test-channels.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const channels = require('../../channels.js');
+const chai = require('chai');
+const expect = chai.expect;
+let event, context;
+
+function isAnObject(result) {
+    console.log("isAnObject")
+    expect(result).to.be.an('object');
+    expect(result.statusCode).to.equal(200);
+    expect(result.body).to.be.an('string');
+}
+
+function isAValidJournal(response) {
+    console.log("isAValidJournal")
+    expect(response).to.be.an('object');
+    expect(response.message).to.be.equal("hello journal");
+}
+
+function isAValidPublisher(response) {
+    console.log("isAValidPublisher")
+    expect(response).to.be.an('object');
+    expect(response.message).to.be.equal("hello publisher");
+}
+
+
+describe('Tests journal', async function () {
```

Comment:
naming should follow the convention. Here it is much easier as it can be a real sentence.